src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/results.html.twig

```diff
@@ -48,7 +48,11 @@
                             {% endif %}
                         </td>
                         <td class="break-long-words">
-                            {{ result.url }}
+                            {% if result.method|upper in ['GET', 'HEAD'] %}
+                                <a href="{{ result.url }}">{{ result.url }}</a>
```

Comment:
URL in header has no `target="blank"`

![webprofiler](https://cloud.githubusercontent.com/assets/1418898/15022588/aade22a6-122c-11e6-8519-af97e8d02a37.png)